_data/team.yml

```diff
@@ -368,3 +368,14 @@
     <p>Richard was switched on to Software Craftsmanship by some excellent agile coaches who taught XP, and through the association of some inspiring Software Craftspeople he has been fortunate to work with. In addition to honing his craft, he loves mentoring and teaching others how to practice TDD and how to write clean pragmatic code.</p>
     <p>Aside from programming, Richard is a keen club runner with the Bedford Harriers, doing all distances from 5K to marathons. He also enjoys cycling, blogging and making music. Richard lives in Northampton with his partner Sarah.</p>
   github: richardjwild
+
+- id: christinaripleygonzalez
+  name: Christina Ripley-Gonzalez
+  image: /assets/img/custom/team/christina_ripley.jpg
+  role: Apprentice
+  fullDescription:
+    <p>Christina has been working as a software developer and team leader in the .Net Ecosystem for a number of years. Enjoying process automation and generaly making others jobs easier.</p>
```

Comment:
"Ecosystem" should not be capitalised.
"Generally" should have double L.
There should be an apostrophe after others, i.e. others'